test/color/test_rgb.py

```diff
@@ -1,126 +1,139 @@
-import pytest
-
 import kornia
-import kornia.testing as utils  # test utils
+import kornia.testing as utils
 from test.common import device_type
-
 import torch
 from torch.autograd import gradcheck
 from torch.testing import assert_allclose
+import pytest
 
 
-class TestBgrToRgb:
+class TestRgbToRgba:
+    def test_rgb_to_rgba(self):
 
-    def test_bgr_to_rgb(self):
+        data = torch.Tensor([[[1., 1.], [1., 1.]], [[2., 2.], [2., 2.]], [[3., 3.], [3., 3.]]])  # 3x2x2
+        aval = 0.4
 
-        # prepare input data
-        data = torch.tensor([[[1., 1.],
-                              [1., 1.]],
+        expected = torch.Tensor([[[1.0000e+00, 1.0000e+00], [1.0000e+00, 1.0000e+00]],
+                                 [[2.0000e+00, 2.0000e+00], [2.0000e+00, 2.0000e+00]],
+                                 [[3.0000e+00, 3.0000e+00], [3.0000e+00, 3.0000e+00]],
+                                 [[1.3563e-19, 1.5686e-03],
+                                  [2.0706e-19, 7.2939e+22]]])  # 4x2x2
```

Comment:
after seeing this example, I'm not quite sure that we have to internally divide by 255 since we are assuming all the time already images in the range of 0-1. /cc @ducha-aiki @dvd42 @priba  thoughts ?